cupy/manipulation/basic.py

```diff
@@ -69,3 +69,61 @@ def _can_memcpy(dst, src):
     f_contiguous = dst.flags.f_contiguous and src.flags.f_contiguous
     return (c_contiguous or f_contiguous) and dst.dtype == src.dtype and \
         dst.size == src.size
+
+
+putmask_kernel = core._kernel.ElementwiseKernel(
```

Comment:
Please add single underscore prefix as it is private function.
```suggestion
_putmask_kernel = core._kernel.ElementwiseKernel(
```